monitoring.tf

```diff
@@ -92,7 +92,7 @@ resource "aws_kms_alias" "backup_sns" {
 
 module "sns_topic" {
   source  = "terraform-aws-modules/sns/aws"
-  version = "~> 3.0"
```

Comment:
lets have some exact(can be latest) version here for module,

can we have this for sns related all resource/modules, so that if notifications are disabled no unusable resources will be created?
```terraform
count             = var.enable_sns_notifications ? 1 : 0
```